.github/workflows/deploy.yml

```diff
@@ -25,6 +25,10 @@ on:
         required: false
         default: "postgres_db"
         type: string
+      apply_network_policy:
```

Comment:
It appears we are not using "users", "spawn-rate", "run-time"

Reply:
ahh good spot! will remove those as they aren't being used anywhere 👍🏾